Add OnBoard to readme (#12)
hackclub/.github: profile/README.md

- Create a game in [🍃 Sprig](https://sprig.hackclub.com) (we'll ship you a [console](https://github.com/hackclub/sprig-hardware)!).

> - Learn how to & make your own PCB with [⚙️ OnBoard](https://hackclub.com/onboard)

Reviewer: ```suggestion
- Make and manufacture your first PCB
 as part of [⚙️ OnBoard](https://hackclub.com/onboard)
```

The and wasn't quite making sense IMO. I don't like the as part of in my suggestion but it makes more sense than with, any ideas?
